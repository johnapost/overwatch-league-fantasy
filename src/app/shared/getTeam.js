// @flow

import teams from "./teams.json";
import type { AbbreviatedNames } from "../components/teamLogo";

export type Team = {
  abbreviatedName: AbbreviatedNames,
  addressCountry: string,
  homeLocation: string,
  id: number,
  name: string,
  owl_division: number,
  primaryColor: string,
  secondaryColor: string
};

export default (id: number): Team => teams.find(item => item.id === id);
