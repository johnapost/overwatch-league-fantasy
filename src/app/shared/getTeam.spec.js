import getTeam from "./getTeam";

describe("getTeam", () => {
  it("should get a team", () => {
    expect(getTeam(4523)).toMatchSnapshot();
  });
});
