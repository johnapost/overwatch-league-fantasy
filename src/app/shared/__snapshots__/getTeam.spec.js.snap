// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getTeam should get a team 1`] = `
Object {
  "abbreviatedName": "DAL",
  "addressCountry": "US",
  "homeLocation": "Dallas, TX",
  "id": 4523,
  "name": "Dallas Fuel",
  "owl_division": 80,
  "primaryColor": "0c2340",
  "secondaryColor": "0072ce",
}
`;
